browser/ios/PluginTests/PluginTests.swift

```diff
@@ -0,0 +1,26 @@
+import XCTest
+@testable import Plugin
+
+class BrowserTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+
+    func testEcho() {
+        // This is an example of a functional test case for a plugin.
+        // Use XCTAssert and related functions to verify your tests produce the correct results.
+
+        let implementation = Browser()
+        let value = "Hello, World!"
+        let result = implementation.echo(value)
+
+        XCTAssertEqual(value, result)
+    }
```

Comment:
```suggestion
```

remove the echo test, we should probably add this to the todo list

Reply:
Oops, good catch.